ui/src/styles/_vars.scss

```diff
@@ -12,6 +12,10 @@
   --pink: hsl(316, 69%, 70%); /* #E77CCA　*/
   --violet: hsl(286, 55%, 53%); /* A943C9 */
 
+  // digital menus card colors
+  --proCard: #9f32b2;
+  --expertCard: #00d7ff;
```

Comment:
we want to use colors within our current color scheme. I know these are different on the code my menu page but can we keep these as either our violet or teal colors?